app/main.py

```diff
@@ -1 +1,64 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills.extend([
+            "JavaScript",
+            "HTML",
+            "CSS"
+        ])
+
+    def create_awesome_web_page(self) -> str:
+        print(f"{self.name} is creating a webpage...")
+        return "<h1>Hello world</h1>"
+
+
+class BackendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills.extend([
+            "Python",
+            "SQL",
+            "Django"
+        ])
+
+    def create_powerful_api(self) -> str:
+        print(f"{self.name} is creating an API...")
+        return "http://127.0.0.1:8000"
+
+
+class AndroidDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills.extend([
+            "Java",
+            "Android studio",
+        ])
+
+    def create_smooth_mobile_app(self) -> str:
+        print(f"{self.name} is creating a mobile app...")
+        return "Ads every three swipes"
+
+
+class FullStackDeveloper(
+    AndroidDeveloper,
+    BackendDeveloper,
+    FrontendDeveloper
+):
+
+    def __init__(self, name: str) -> None:
+        super(FullStackDeveloper, self).__init__(name)
+        self.skills = ['CSS', 'Django', 'HTML', 'JavaScript', 'Python', 'SQL']
```

Comment:
You don't have to assing `self.skills` here. It is already done in the `super()` calls.